projects/plugins/jetpack/_inc/client/state/connection/reducer.js

```diff
@@ -144,11 +145,25 @@ export const requests = ( state = connectionRequests, action ) => {
 	}
 };
 
+export const hasSeenWCConnectionModalReducer = (
+	state = window.Initial_State.hasSeenWCConnectionModal || false,
+	action
+) => {
+	switch ( action.type ) {
+		case JETPACK_CONNECTION_HAS_SEEN_WC_CONNECTION_MODAL:
+			return true;
```

Comment:
I'm most likely missing something, but shouldn't a reducer like this usually return a state object?

Reply:
In this case, this is just a slice of state (`connection.hasSeenWCConnectionModal`) and it slice can be either `true` or `false`.

Reply:
Gotcha, thanks.